### DISCUSSION TITLE
希望增加文本简繁体转换功能

### DISCUSSION THREAD

**Discussion by fxph35** [he3-app/he3-feedback#55] [Ideas]
有时候有些地方需要繁体字，每次只能去搜索引擎转换。

**Comment by hongchaodeng** [comment]
这个功能跟开发者工具偏得有点远哈 😅

等我们开放体系出来后看看能不能有社区同学来贡献

**Reply by fxph35** [reply]
不不不，这个不远，比如多语言App，香港澳门需要繁体字的。😂

**Reply by hongchaodeng** [reply]
好的。这个整体的翻译功能到时候我们考虑通过集成云服务 (ChatGPT, Google Translate) 这样来提供
